content/it/docs/concepts/overview/kubernetes-api.md

```diff
@@ -1,5 +1,7 @@
 ---
-title: Le API di Kubernetes 
+title: Le API di Kubernetes
+description: >
+  Le API di Kubernetes ti permettono di interrogare e manipolare lo stato degli oggetti in Kubernetes. Il core del piano di controllo di Kubernetes è l'API server e le API HTTP che esso espone. Ogni entità o componente che si interfaccia con il cluster (gli utenti, le singole parti del tuo cluster, i componenti esterni), comunica attraverso l'API server.
```

Comment:
* I would substitute "core" for "cuore", since "core" is not much used in Italian in this way
* "Piano di controllo" which I believe would be the translation of "Control Plane" is not a term used in Italian in this meaning. Please, let's keep the English version (as is in the rest of the file)